packages/cli/src/workflows/workflows.services.ts

```diff
@@ -0,0 +1,32 @@
+import { FindOneOptions, ObjectLiteral } from 'typeorm';
+import { Db } from '..';
+import { SharedWorkflow } from '../databases/entities/SharedWorkflow';
+import { User } from '../databases/entities/User';
+
+export class WorkflowsService {
+	static async getSharing(
+		user: User,
+		workflowId: number | string,
+		relations: string[] | undefined = ['workflow'],
+		{ allowGlobalOwner } = { allowGlobalOwner: true },
```

Comment:
Rather odd to require the caller to pass in `undefined` to use the default relation.

```suggestion
		options = { relations: ['workflow'], allowGlobalOwner: true },
```

Reply:
Good catch, thanks. I've copied it from the `credentials.services.ts` and did not see this.